cloudflare/resource_cloudflare_access_rule.go

```diff
@@ -258,3 +260,47 @@ func configurationDiffSuppress(k, old, new string, d *schema.ResourceData) bool
 
 	return false
 }
+
+func validateAccessRuleConfiguration(v interface{}, k string) (warnings []string, errors []error) {
+	config := v.(map[string]interface{})
+
+	target := config["target"].(string)
+	value := config["value"].(string)
+
+	switch target {
+	case "ip_range":
+		return validateAccessRuleConfigurationIPRange(value)
+	default:
+	}
+
+	return warnings, errors
+}
+
+func validateAccessRuleConfigurationIPRange(v string) (warnings []string, errors []error) {
+	ip, ipNet, err := net.ParseCIDR(v)
+	if err != nil {
+		errors = append(errors, fmt.Errorf("failed to parse value as CIDR: %v", err))
+		return warnings, errors
+	}
+
+	if ipNet == nil {
+		errors = append(errors, fmt.Errorf("ip_range must hold a range"))
+		return warnings, errors
+	}
+
+	if ip.To4() != nil {
+		ones, _ := ipNet.Mask.Size()
+		if ones != 24 && ones != 32 {
```

Comment:
Shouldn't this be `if ones != 16 && ones != 24 {`? From the docs

> Limited to /16 and /24 for IPv4 and /32, /48 and /64 for IPv6.

Reply:
Yes! Good catch.